pyproject.toml

```diff
@@ -59,13 +59,9 @@ select = [
   "RUF",
   "I",
 ]
-ignore = [
-  "ANN101", # no type for self
-  "ANN102", # no type for cls
-]
-unfixable = ["B", "RUF"]
```

Comment:
Why is this no longer needed?

Reply:
![CleanShot 2025-02-05 at 22 31 26](https://github.com/user-attachments/assets/25903e53-28a2-45bb-a7f1-03daa2690221)